oxenmq/formattable.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <fmt/format.h>
```

Comment:
This should also `#include "connections.h"` and likewise for `auth.h` (otherwise the types in the specializations won't be available).